mne/decoding/receptive_field.py

```diff
@@ -17,17 +17,18 @@
 class ReceptiveField(BaseEstimator):
     """Fit a receptive field model.
 
-    This allows you to fit a model using time-lagged input features. For
-    example, a spectro- or spatio-temporal receptive field (STRF).
+    This allows you to fit an encoding model (stimulus to brain) using
+    time-lagged input features. For example, a spectro- or spatio-temporal
+    receptive field (STRF).
 
     Parameters
     ----------
     tmin : float
         The starting lag, in seconds (or samples if ``sfreq`` == 1).
-        Negative values correspond to times in the past.
+        Negative values correspond to times in the future.
```

Comment:
Eeww. I know this is an attempt to clarify, but I think it makes things more confusing. It sounds like we are flipping when we're not.

Reply:
hmmm - what would be the wording that would clarify it in your mind? We need to come up with something that is satisfying to both people who intuitively think negative == past, and those who think negative == future...

Reply:
Maybe say something like "For a causal system, the encoding model would have significant non-zero values only at positive lags."

Reply:
what if we said just as a note in the docstring: "Note that lags point backwards in time, so positive lags correspond to timepoints in the past, while negative lags correspond to timepoints in the future."

Reply:
So compromise, don't say anything about neg/pos here (because now we match convention anyway) and add:
```
Notes
-----
For a causal system, the encoding model would have significant
non-zero values only at positive lags. In other words, lags point
backwards in time relative to the input, so positive lags correspond
to previous time samples, while negative lags correspond to future
time samples.
```

Reply:
+1 I can merge quickly if you wanna push it to this branch

Reply:
also I think the examples will clarify this (maybe we should add a link?)